lib/resources/r4/charge_item_modify.yaml

```diff
@@ -0,0 +1,437 @@
+                    {
+                      "valueCoding": {
+                        "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/14002",
+                        "code": "615214"
+                      },
+                      "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/bill-code-schedule"
+                    }
+                  ],
+                  "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/CodeSystem/BillCodes-CPT",
+                  "code": "10081"
+                }
+              ]
+            },
+            "url": "code"
+          },
+          {
+            "valueUnsignedInt": 1,
+            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/priority"
+          }
+        ],
+        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/procedure-code"
+      }
+    note: >
+      <ul>
+        <li>The coding field must contain only one coding from the CPT, HCPCS, or PROCCODE proprietary Bill Codes Types systems</li>
+        <li>Must be of a lower bill code tier than the ChargeItem.code field if provided</li>
+        <li>Both code and priority sub extensions are required</li>
+      </ul>
+    url: https://fhir-ehr.cerner.com/r4/StructureDefinition/procedure-code?_format=json
+
+  - name: revenue-code
+    required: 'No'
+    type: Extension
+    description: The type of revenue or cost center providing the product and/or service.
+    action: modify
+    example: |
+      {
+        "valueCodeableConcept": {
+          "coding": [
+            {
+              "extension": [
+                {
+                  "valueCoding": {
+                    "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/14002",
+                    "code": "615217"
+                  },
+                  "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/bill-code-schedule"
+                }
+              ],
+              "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/CodeSystem/BillCodes-REVENUE",
+              "code": "0024"
+            }
+          ]
+        },
+        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/revenue-code"
+      }
+    note: >
+      <ul>
+        <li>The coding field must contain only one coding from the REVENUE proprietary Bill Codes Types system</li>
+      </ul>
+    url: https://fhir-ehr.cerner.com/r4/StructureDefinition/revenue-code?_format=json
+
+  - name: transactionId
+    required: 'Yes'
+    type: uuid
+    description: A UUID to uniquely identify the modify operation.
+    action: modify
+    example: |
+      {
+        "parameter": [
+          {
+            "name": "transactionId",
+            "valueUuid": "urn:uuid:aa7623a0-0c7b-11ec-82a8-0242ac130003"
+          }
+        ]
+      }
+    note: >
+      <ul>
+        <li>The value must be in the form of `valueUuid`</li>
+        <li>Only one value can be defined</li>
```

Comment:
In reference to this comment: https://github.com/cerner/fhir.cerner.com/pull/780/files#r766187644

It looks like this note is actually meant to specify that only one transactionId parameter is provided (since value is defined as 0..1 by FHIR). Let's update this wording to 'Only one `transactionId` parameter may be provided' and add it to the chargeItem parameter as well.

cc. @alexbeckner1

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/780/commits/0e5c3b242a28011c947985372687f1784aea06ba#diff-3c565d9c41e2bb3d417efe63a774683e72b07cd38f2ff54a695c695999f29938R63